cwl/src/test/scala/wdl4s/cwl/CwlWorkflowWomSpec.scala

```diff
@@ -49,7 +48,7 @@ class CwlWorkflowWomSpec extends FlatSpec with Matchers {
               map(_.select[CommandLineTool].get).
               value.
               unsafeRunSync
-      wom <-  clt.womExecutable.toEither
+      wom <-  clt.womExecutable
```

Comment:
Extra space persists

Reply:
ah shoot!  This CWL subproject is the first to get auto-formatted, so stand back!